### PR TITLE
Git HW and project (IRSI)

### PR DESCRIPTION
**Pull request by Xstunter**
HW Git was comleted.
Project about insert, swap,raplace, immposible +

### CODE REVIEW

File: Lesson_3/Lesson_3/Program.cs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Console.Write("Input nocorrect word:");
+string T = Console.ReadLine();*/
+
+Console.WriteLine(Func("nice", "niece"));
+Console.WriteLine(Func("test", "tent"));
+Console.WriteLine(Func("form", "from"));
+Console.WriteLine(Func("o", "odd"));
+
+string Func(string S, string T)
```
Comment (petrochernykh) [inline Lesson_3/Lesson_3/Program.cs:14]: This method name is not describe what this method will do

File: Lesson_3/Lesson_3/Program.cs [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                    else
+                    {
+                        return "IMPOSSIBLE";
+                    }
+                }
+            }
+            //end
+        } 
+    }
+    else if (S==T)
```
Comment (petrochernykh) [inline Lesson_3/Lesson_3/Program.cs:81]: Just 'else' will be enought

File: Lesson_3/Lesson_3/Program.cs [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+string Func(string S, string T)
+{
+    if (S != T)
+    {
+        char correctChar = default(char), nocorrectChar = default(char);
+
+        int count = 0;
+
+        for (int i = 0; i < S.Length; i++)
```
Comment (petrochernykh) [inline Lesson_3/Lesson_3/Program.cs:22]: What if this is word with 2 different characters like **nice** and **node**

**Comment by Xstunter** [comment]
Output is IMPOSSIBLE

15 жовтня 2023, 23:54:05, від "petrochernykh" ***@***.***>:


@petrochernykh commented on this pull request.
In Lesson_3/Lesson_3/Program.cs:
> +string T = Console.ReadLine();*/
+
+Console.WriteLine(Func("nice", "niece"));
+Console.WriteLine(Func("test", "tent"));
+Console.WriteLine(Func("form", "from"));
+Console.WriteLine(Func("o", "odd"));
+
+string Func(string S, string T)
+{
+    if (S != T)
+    {
+        char correctChar = default(char), nocorrectChar = default(char);
+
+        int count = 0;
+
+        for (int i = 0; i < S.Length; i++) 
What if this is word with 2 different characters like nice and node
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>

--=-uw2zXge9vz24gKZGzu4F
Content-Type: text/html; charset="UTF-8"
Content-Transfer-Encoding: base64

PGh0bWw+PGJvZHk+PHNwYW4gc3R5bGU9ImRpc3BsYXk6YmxvY2s7IiBjbGFzcz0ieGZtXzcxNzQ2
MDA1Ij48ZGl2PjxzcGFuIHN0eWxlPSJmb250LXNpemU6MTJwdDtsaW5lLWhlaWdodDoxNHB0O2Zv
bnQtZmFtaWx5OkFyaWFsOyIgY2xhc3M9InhmbWMxIj5PdXRwdXQgaXMgSU1QT1NTSUJMRTwvc3Bh
bj48YnIvPjwvZGl2Pg0KPGRpdj48YnIvPjwvZGl2Pg0KPGRpdj48aT48c3BhbiBzdHlsZT0iZm9u
dC1zaXplOjEwcHQ7bGluZS1oZWlnaHQ6MTJwdDsiPjxzcGFuIHN0eWxlPSJmb250LWZhbWlseTpB
cmlhbDsiPjE1INC20L7QstGC0L3RjyAyMDIzLCAyMzo1NDowNSwg0LLRltC0DQogICAgInBldHJv
Y2hlcm55a2giICZsdDs8L3NwYW4+PGEgaHJlZj0ibWFpbHRvOm5vdGlmaWNhdGlvbnNAZ2l0aHVi
LmNvbSIgdGFyZ2V0PSJfYmxhbmsiPjxzcGFuIHN0eWxlPSJmb250LWZhbWlseTpBcmlhbDsiPm5v
dGlmaWNhdGlvbnNAZ2l0aHViLmNvbTwvc3Bhbj48L2E+PHNwYW4gc3R5bGU9ImZvbnQtZmFtaWx5
OkFyaWFsOyI+Jmd0OzoNCjwvc3Bhbj48L3NwYW4+PC9pPjwvZGl2Pg0KPGRpdj48YnIvPjwvZGl2
Pg0KPGJsb2NrcXVvdGUgc3R5bGU9ImJvcmRlci1sZWZ0OjFweCBzb2xpZCAjY2NjY2NjO21hcmdp
bjowcHggMHB4IDBweCAwLjhleDtwYWRkaW5nLWxlZnQ6MWV4OyI+DQo8ZGl2IHN0eWxlPSJkaXNw
bGF5OmJsb2NrOyI+DQo8ZGl2PjwvZGl2Pg0KPGRpdj48Yj5AcGV0cm9jaGVybnlraDwvYj4gY29t
bWVudGVkIG9uIHRoaXMgcHVsbCByZXF1ZXN0LjwvZGl2Pg0KPGhyLz48ZGl2PkluIDxhIGhyZWY9
Imh0dHBzOi8vZ2l0aHViLmNvbS9Yc3R1bnRlci9BTGV2ZWwvcHVsbC8xI2Rpc2N1c3Npb25fcjEz
NTk5NDgzMzQiIHRhcmdldD0iX2JsYW5rIiByZWw9Im5vcmVmZXJyZXIgbm9vcGVuZXIiPkxlc3Nv
bl8zL0xlc3Nvbl8zL1Byb2dyYW0uY3M8L2E+OjwvZGl2Pg0KPHByZSBzdHlsZT0ibWFyZ2luOjVw
eCAwOyI+Jmd0OyArc3RyaW5nIFQgPSBDb25zb2xlLlJlYWRMaW5lKCk7Ki8NCisNCitDb25zb2xl
LldyaXRlTGluZShGdW5jKCJuaWNlIiwgIm5pZWNlIikpOw0KK0NvbnNvbGUuV3JpdGVMaW5lKEZ1
bmMoInRlc3QiLCAidGVudCIpKTsNCitDb25zb2xlLldyaXRlTGluZShGdW5jKCJmb3JtIiwgImZy
b20iKSk7DQorQ29uc29sZS5Xcml0ZUxpbmUoRnVuYygibyIsICJvZGQiKSk7DQorDQorc3RyaW5n
IEZ1bmMoc3RyaW5nIFMsIHN0cmluZyBUKQ0KK3sNCisgICAgaWYgKFMgIT0gVCkNCisgICAgew0K
KyAgICAgICAgY2hhciBjb3JyZWN0Q2hhciA9IGRlZmF1bHQoY2hhciksIG5vY29ycmVjdENoYXIg
PSBkZWZhdWx0KGNoYXIpOw0KKw0KKyAgICAgICAgaW50IGNvdW50ID0gMDsNCisNCisgICAgICAg
IGZvciAoaW50IGkgPSAwOyBpICZsdDsgUy5MZW5ndGg7IGkrKykgDQo8L3ByZT4NCjxkaXY+V2hh
dCBpZiB0aGlzIGlzIHdvcmQgd2l0aCAyIGRpZmZlcmVudCBjaGFyYWN0ZXJzIGxpa2UgPGI+bmlj
ZTwvYj4gYW5kIDxiPm5vZGU8L2I+PC9kaXY+DQo8ZGl2PjxzcGFuIHN0eWxlPSJmb250LXNpemU6
c21hbGw7Y29sb3I6IzY2NjsiPuKAlDxici8+UmVwbHkgdG8gdGhpcyBlbWFpbCBkaXJlY3RseSwg
PGEgaHJlZj0iaHR0cHM6Ly9naXRodWIuY29tL1hzdHVudGVyL0FMZXZlbC9wdWxsLzEjcHVsbHJl
cXVlc3RyZXZpZXctMTY3ODg3MDAwNiIgdGFyZ2V0PSJfYmxhbmsiIHJlbD0ibm9yZWZlcnJlciBu
b29wZW5lciI+dmlldyBpdCBvbiBHaXRIdWI8L2E+LCBvciA8YSBocmVmPSJodHRwczovL2dpdGh1
Yi5jb20vbm90aWZpY2F0aW9ucy91bnN1YnNjcmliZS1hdXRoL0E0S01LWkZPQVVHMlVGVlBSNkU2
NkZMWDdSRU9aQU5DTkZTTTZBQUFBQUE2QkRYN1dNIiB0YXJnZXQ9Il9ibGFuayIgcmVsPSJub3Jl
ZmVycmVyIG5vb3BlbmVyIj51bnN1YnNjcmliZTwvYT4uPGJyLz5Zb3UgYXJlIHJlY2VpdmluZyB0
aGlzIGJlY2F1c2UgeW91IGF1dGhvcmVkIHRoZSB0aHJlYWQuPGltZyBzcmM9Imh0dHBzOi8vZ2l0
aHViLmNvbS9ub3RpZmljYXRpb25zL2JlYWNvbi9BNEtNS1pFRFZWUkRDN0NGVUw3TVZTVFg3UkVP
WkE1Q05GU002QUFBQUFBNkJEWDdXT1dHRzMzTk5WU1c0NUM3T1I0WEFaTlJLQjJXWTNDU01WWVhL
WkxUT1JKR0s1VEpNVjMyVVkzUE5WV1dLM1RVTDVVV0pUVEVDR0M3TS5naWYiIGhlaWdodD0iMSIg
d2lkdGg9IjEiIGFsdD0iIi8+PC9zcGFuPjxzcGFuIHN0eWxlPSJmb250LXNpemU6MDtjb2xvcjp0
cmFuc3BhcmVudDtkaXNwbGF5Om5vbmU7b3ZlcmZsb3c6aGlkZGVuO3dpZHRoOjA7aGVpZ2h0OjA7
bWF4LXdpZHRoOjA7bWF4LWhlaWdodDowOyI+TWVzc2FnZSBJRDogJmx0O1hzdHVudGVyL0FMZXZl
bC9wdWxsLzEvcmV2aWV3LzE2Nzg4NzAwMDZAZ2l0aHViLmNvbSZndDs8L3NwYW4+PC9kaXY+DQo8
L2Rpdj4NCjwvYmxvY2txdW90ZT48L3NwYW4+PC9ib2R5PjwvaHRtbD4NCg==

--=-uw2zXge9vz24gKZGzu4F--

**Pull request merged**